### ISSUE TITLE
Wrong compile with imports

### ISSUE THREAD

**Issue by lviggiani** [codehz/deno-pbf#2]
Consider the attached two `.proto` files.
I compile them with the following code:

```typescript
import * as path from "https://deno.land/std@0.176.0/path/mod.ts";
import { compile, parse } from "https://deno.land/x/pbf@v0.1.2/mod.ts";
import { printf } from "https://deno.land/std@0.175.0/fmt/printf.ts";

const srcDir = "./proto";
const tgtDir = "./classes/protobuf/generated";

await Deno.remove(tgtDir, { recursive: true }).catch(e => {});

const compileFolder = async (srcFolder: string, tgtFolder: string) => {
    await Deno.mkdir(tgtFolder, { recursive: true }).catch(e => {});
    for await (const dirEntry of Deno.readDir(srcFolder)) {
        if (dirEntry.isDirectory){
            await compileFolder(
                path.join(srcFolder, dirEntry.name),
                path.join(tgtFolder, dirEntry.name));
        } else {
            const src = path.join(srcFolder, dirEntry.name);
            const dest = dirEntry.name.replace(/\.proto$/, ".generated.ts");
            printf(`Compiling ${src}...`);
            const proto = await Deno.readTextFile(src);
            const ts = compile(parse(proto), { basedir: srcFolder});
            await Deno.writeTextFile(path.join(tgtFolder, dest), ts);
            printf("done\n");
        }
    }
}

await compileFolder(srcDir, tgtDir);

console.log("completed");
```

It creates a wrong output for `msgOwnship.generated.ts`:

```typescript
  function _readField(tag: number, obj: MsgOwnship, pbf: Pbf) {
    if (tag === 1) {
      obj.type = MsgId.messageType.read(pbf, pbf.readVarint() + pbf.pos);  // <<<< ERR: MsgId.messageType.read method not found
```

and 

```typescript
  export function write(obj: Partial<MsgOwnship>, pbf: Pbf) {
    if (obj.type != null) {
      pbf.writeMessage(1, MsgId.messageType.write, obj.type); // <<<< ERR: MsgId.messageType.write method not 
    }
```
[proto.zip](https://github.com/codehz/deno-pbf/files/10687237/proto.zip)

**Comment by codehz** [comment]
I'm looking into it, thanks a lot for reporting this issue

**Comment by codehz** [comment]
should be fixed in 921928025ded07bf87f8adbf5e4e4afc63228ec3 (released in v0.2.0)
If you still have problems, feel free to reopen

**Comment by lviggiani** [comment]
I confirm that v0.2.0 fixes the issue. Thanks!

**Comment by lviggiani** [comment]
FYI, this also fixes issue #1
